src/Avalonia.NameGenerator/Avalonia.NameGenerator.csproj

```diff
@@ -1,20 +1,27 @@
-<Project Sdk="Microsoft.NET.Sdk">
-    <PropertyGroup>
-        <TargetFramework>netstandard2.0</TargetFramework>
-        <LangVersion>preview</LangVersion>
-        <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-        <IncludeBuildOutput>false</IncludeBuildOutput>
-        <PackageId>XamlNameReferenceGenerator</PackageId>
-    </PropertyGroup>
-    <ItemGroup>
-        <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.8.0" PrivateAssets="all" />
-        <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.2" PrivateAssets="all" />
-    </ItemGroup>
-    <ItemGroup>
-        <Compile Link="XamlX\filename" Include="../../external/XamlX/src/XamlX/**/*.cs" />
-        <Compile Remove="../../external/XamlX/src/XamlX/**/SreTypeSystem.cs" />
-    </ItemGroup>
-    <ItemGroup>
-        <None Include="$(OutputPath)\$(AssemblyName).dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
-    </ItemGroup>
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
```

Comment:
Thanks for the PR! nit: Worth either switching back to using spaces or adding a `.editorconfig` with `indent_size` setting similar to this one https://github.com/zkSNACKs/WalletWasabi/blob/master/.editorconfig#L14-L16 (the updated GitHub UI defaults to 8 spaces per tab and this might hurt readability)